prisma/schema.prisma

```diff
@@ -150,32 +151,32 @@ model Attendee {
   email     String
   name      String
   timeZone  String
-  booking   Booking? @relation(fields: [bookingId], references: [id])
+  booking   Booking? @relation(fields: [bookingId], references: [id], onDelete: Cascade, onUpdate: Cascade)
   bookingId Int?
 }
 
 enum BookingStatus {
-  CANCELLED @map("cancelled")
-  ACCEPTED  @map("accepted")
-  REJECTED  @map("rejected")
-  PENDING   @map("pending")
+  CANCELLED  @map("cancelled")
+  ACCEPTED   @map("accepted")
+  REJECTED   @map("rejected")
+  PENDING    @map("pending")
 }
 
 model DailyEventReference {
-  id         Int         @id @default(autoincrement())
-  dailyurl   String      @default("dailycallurl")
-  dailytoken String      @default("dailytoken")
-  booking    Booking?    @relation(fields: [bookingId], references: [id])
-  bookingId  Int?
+  id         Int      @id @default(autoincrement())
+  dailyurl   String   @default("dailycallurl")
+  dailytoken String   @default("dailytoken")
+  booking    Booking? @relation(fields: [bookingId], references: [id], onDelete: Cascade, onUpdate: Cascade)
+  bookingId  Int?     @unique
 }
 
 model Booking {
   id          Int                @id @default(autoincrement())
   uid         String             @unique
-  user        User?              @relation(fields: [userId], references: [id])
+  user        User?              @relation(fields: [userId], references: [id], onDelete: Cascade, onUpdate: Cascade)
   userId      Int?
   references  BookingReference[]
-  eventType   EventType?         @relation(fields: [eventTypeId], references: [id])
+  eventType   EventType?         @relation(fields: [eventTypeId], references: [id], onDelete: Cascade, onUpdate: Cascade)
```

Comment:
Deletes Booking if EventType is deleted --- maybe this should be different - possible set null instead